// A simple exception handling example.

#include <iostream>
using namespace std;

int main() {



}